6.0.0.md

```diff
@@ -0,0 +1,42 @@
+# Parse Server 6 Migration Guide <!-- omit in toc -->
+
+This document only highlights specific changes that require a longer explanation. For a full list of changes in Parse Server 6 please refer to the [changelog](https://github.com/parse-community/parse-server/blob/alpha/CHANGELOG.md).
+
+---
+
+- [Asynchronous Initialization](#asynchronous-initialization)
+
+---
+
+## Asynchronous Initialization
+
+The import and initialization syntax has been simplified with more intuitive naming and structure.
+
+Previously, it was possible to mount Parse Server before it was fully started up and ready to receive requests. This could result in undefined behavior, such as Parse Objects could be saved before Cloud Code was registered. To prevent this, Parse Server 6 requires to be started asynchronously before being mounted.
+
+*Parse Server 5:*
+```js
+// 1. Import Parse Server
+const { ParseServer } = require('parse-server');
+
+// 2. Create a Parse Server instance as express middleware
+const server = new ParseServer(config);
+
+// 3. Mount express middleware
+app.use("/parse", server);
+```
+
+*Parse Server 6:*
+```js
+// 1. Import Parse Server
+const ParseServer = require('parse-server');
```

Comment:
@mtrezza You can still use the named export of Parse Server, there are no changes there

```suggestion
const { ParseServer } = require('parse-server');
```

Reply:
So which one is correct:

```js
const ParseServer = require('parse-server');
const server = new ParseServer(config);
```

or

```js
const  { ParseServer } = require('parse-server');
const server = new ParseServer(config);
```

Reply:
Both return parse server instances in PS6, which do you think should be preferred?

Reply:
Why do we need both? I think ambiguity should be avoided in preference to providing one clear path.

Reply:
@dblythy what do you think about the corrected migration guide? It has an extra chapter regarding import statement.

Reply:
Looks good